ui/css/treeherder-details-panel.css

```diff
@@ -307,8 +307,17 @@ ul.failure-summary-list li .btn-xs {
   background: #ffffff;
 }
 
-.failure-summary-bugs {
-  padding: 0 0 0 18px;
+.failure-summary-bugs-title {
+  font-size: 12px;
+}
+
+.failure-summary-bugs-container {
+  /* These styles don't use the bootstrap system as they are creating a vertical
+   * line underneath the bug icon. When only using bootstrap classes, this vertical line
+   * is not aligned with the bug icon. */
+  margin: 0 11px;
+  padding: 10px 15px;
+  border-left: 2px solid #0004;
```

Comment:
This rule is creating the vertical line:

<img width="186" alt="image" src="https://user-images.githubusercontent.com/1588648/83075637-d0824600-a039-11ea-884a-81a60124384e.png">

Bootstrap styles weren't aligned here, and the border was very thin. I wrote some justification comments about this compromise.